CONTRIBUTING.md

```diff
@@ -34,6 +34,29 @@ Example on macOS using Homebrew:
     brew install imagemagick
     bundle exec rspec
 
+Example setup on Ubuntu:
+
+```bash
+bundle install
+yarn install
+# install imagemagick:
+sudo apt update -y && sudo apt install imagemagick -y
+# install dependencies for each appraisal:
+bundle exec appraisal install
+# install phantomjs
+https://gist.githubusercontent.com/pcai/af7cf1fecc0b465cbeeab306749d867d/raw/6e3cef3184af8443662d83f897bb9b2d5d69cd1e/install_phantomjs.sh
+# install google chrome:
+sudo apt update -y && wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && sudo apt install ./google-chrome-stable_current_amd64.deb -y
+# precompile assets in the dummy app:
+cd spec/dummy_app && yarn install && yarn build && yarn build:css && cd -
+```
+
+Then you will be able to run the specs:
+
+```bash
+BUNDLE_GEMFILE=gemfiles/rails_7.0.gemfile bundle exec rspec
```

Comment:
This is not the expected way to use appraisal.
https://github.com/thoughtbot/appraisal#usage